.travis.yml

```diff
@@ -0,0 +1,108 @@
+# Travis CI (MIT License) configuration file for the Twenty Sixteen theme.
+# @link https://travis-ci.org/
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Declare versions of PHP to use. Use one decimal max.
+php:
+  # aliased to a recent 5.5.x version
+  - "5.5"
+  # aliased to a recent 5.4.x version
+  - "5.4"
+  # aliased to a recent 5.3.x version
+  - "5.3"
+  # Current $required_php_version for WordPress: 5.2.4
+  - "5.2"
+
+# Declare which versions of WordPress to test against.
+# Also declare whether or not to test in Multisite.
+env:
+  # Trunk
+  # @link https://github.com/WordPress/WordPress
+  - WP_VERSION=master WP_MULTISITE=0
+  - WP_VERSION=master WP_MULTISITE=1
+  # WordPress 4.3
+  # @link https://github.com/WordPress/WordPress/tree/4.3-branch
+  - WP_VERSION=4.3 WP_MULTISITE=0
+  - WP_VERSION=4.3 WP_MULTISITE=1
+  # WordPress 4.2
+  # @link https://github.com/WordPress/WordPress/tree/4.2-branch
+  - WP_VERSION=4.2 WP_MULTISITE=0
+  - WP_VERSION=4.2 WP_MULTISITE=1
+
+# Declare 5.6 beta in test matrix.
+# @link https://buddypress.trac.wordpress.org/ticket/5620
+# @link http://docs.travis-ci.com/user/build-configuration/
+matrix:
+  include:
+    - php: 5.6
+      env: WP_VERSION=master
+  allow_failures:
+    - php: 5.6
+  fast_finish: true
+
+# Use this to prepare the system to install prerequisites or dependencies.
+# e.g. sudo apt-get update.
+# Failures in this section will result in build status 'errored'.
+# before_install:
+
+# Use this to prepare your build for testing.
+# e.g. copy database configurations, environment variables, etc.
+# Failures in this section will result in build status 'errored'.
+before_script:
+  # Set up WordPress installation.
+  - export WP_DEVELOP_DIR=/tmp/wordpress/
+  - mkdir -p $WP_DEVELOP_DIR
+  # Use the Git mirror of WordPress.
+  - git clone --depth=1 --branch="$WP_VERSION" git://develop.git.wordpress.org/ $WP_DEVELOP_DIR
+  # Set up theme information.
+  - theme_slug=$(basename $(pwd))
+  - theme_dir=$WP_DEVELOP_DIR/src/wp-content/themes/$theme_slug
+  - cd ..
+  - mv $theme_slug $theme_dir
+  # Set up WordPress configuration.
+  - cd $WP_DEVELOP_DIR
+  - echo $WP_DEVELOP_DIR
+  - cp wp-tests-config-sample.php wp-tests-config.php
+  - sed -i "s/youremptytestdbnamehere/wordpress_test/" wp-tests-config.php
+  - sed -i "s/yourusernamehere/root/" wp-tests-config.php
+  - sed -i "s/yourpasswordhere//" wp-tests-config.php
+  # Create WordPress database.
+  - mysql -e 'CREATE DATABASE wordpress_test;' -uroot
+  # Install CodeSniffer for WordPress Coding Standards checks.
+  - git clone https://github.com/squizlabs/PHP_CodeSniffer.git php-codesniffer
+  # Install WordPress Coding Standards.
+  - git clone https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git wordpress-coding-standards
```

Comment:
@philiparthurmoore since `develop` is set to be the default branch for WordPress-Coding-Standards, this should probably clone explicitly checking out `master` for the latest stable release.

```bash
git clone -b master https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git wordpress-coding-standards
```

Also, you could get a speed boost by grabbing the tarball instead:

```bash
mkdir wordpress-coding-standards && curl -L https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/archive/master.tar.gz | tar xz --strip-components=1 -C wordpress-coding-standards
```

/via https://github.com/xwp/wp-dev-lib/blob/3a17a561580ef73bc63466ab62893535ebad0666/travis.before_script.sh#L52